src/main/java/com/redhat/cloud/common/clowder/configsource/handlers/QuarkusRedisClowderPropertyHandler.java

```diff
@@ -0,0 +1,34 @@
+package com.redhat.cloud.common.clowder.configsource.handlers;
+
+import com.redhat.cloud.common.clowder.configsource.ClowderConfig;
+import com.redhat.cloud.common.clowder.configsource.ClowderConfigSource;
+
+public class QuarkusRedisClowderPropertyHandler extends ClowderPropertyHandler {
+    private static final String QUARKUS_REDIS = "quarkus.redis.";
+
+    public QuarkusRedisClowderPropertyHandler(ClowderConfig clowderConfig) {
+        super(clowderConfig);
+    }
+
+    @Override
+    public boolean handles(String property) {
+        return property.startsWith(QUARKUS_REDIS);
+    }
+
+    @Override
+    public String handle(String property, ClowderConfigSource configSource) {
+        if (clowderConfig.inMemoryDb == null) {
+            throw new IllegalStateException("No inMemoryDb section found");
+        }
+
+        String sub = property.substring(QUARKUS_REDIS.length());
+
+        return switch (sub) {
+            case "hosts" -> "redis://" + clowderConfig.inMemoryDb.hostname + ":" + clowderConfig.inMemoryDb.port;
```

Comment:
Could you please check if [TLS](https://quarkus.io/guides/redis-reference#use-tls) will be supported in our environments? If so, we'll need `rediss://` instead of `redis://` here.